### PR TITLE
Addon-a11y: Move stories into addon

### PR DESCRIPTION
**Pull request by ndelangen**
- [x] add generic stories for addon-viewports in `code/addons/a11y/template/stories`
- [x] remove stories for a11y everywhere else

### CODE REVIEW

**Review by tmeasday — COMMENTED** [review]
I feel like there are a lot of stories deleted here that we need to figure out how to reimplement somehow. Obviously a11y is tricky as the stories depend on HTML structure, is this blocked by creating an `Html` component?

File: code/examples/official-storybook/stories/addon-a11y/highlight.stories.js
Comment (ndelangen) [reply code/examples/official-storybook/stories/addon-a11y/highlight.stories.js:29]: This is actually using the highlight addon, it does not coverage a11y-addon at all.

**Comment by ndelangen** [comment]
@tmeasday I argued to @shilman that we should test the things that matter, and are storybook implemented. I think there's no point in testing axe internals.

But yes, I do think this is blocked for now, and likely should not be merged, or if we do, we should create follow-up tasks to ensure we get the proper coverage again.

File: code/examples/official-storybook/stories/addon-a11y/form.stories.js
Comment (ndelangen) [reply code/examples/official-storybook/stories/addon-a11y/form.stories.js:19]: Yes, using the Html component, I think.

File: code/examples/official-storybook/stories/addon-a11y/base-button.stories.js
Comment (ndelangen) [reply code/examples/official-storybook/stories/addon-a11y/base-button.stories.js:23]: I don't think we do. It's testing a failure of Axe to detect a problem. It'd be great if it worked, but it's not up to Storybook to test it, I think.

File: code/examples/official-storybook/stories/addon-a11y/base-button.stories.js
Comment (ndelangen) [reply code/examples/official-storybook/stories/addon-a11y/base-button.stories.js:30]: What is the point of this story?

I guess the point is to test the `rerun-tests` button in the addon?
I'm not sure this is valuable as a story. I'm inclined to say it isn't useful.

File: code/examples/official-storybook/stories/addon-a11y/form.stories.js
Comment (ndelangen) [reply code/examples/official-storybook/stories/addon-a11y/form.stories.js:19]: added!

**Comment by ndelangen** [comment]
@tmeasday quick question for you about the sandbox script.
So I have the stories show up, but the a11y panel does not show up in the published storybook, and I'm struggling to find why it's not showing up... can you tell me where to look?

https://62f0fd0fd0ff1407689dfeca-xpfmtcuisw.chromatic.com/?path=/story/addons-a11y-parameters--checks&globals=backgrounds.grid:false

I can only see 3: Controls, Actions & Interactions..
But I do not see A11y panel.

**Comment by tmeasday** [comment]
@ndelangen I think the issue is you added `a11y` to the set of "default" addons whose stories are linked by the sandbox. But this is supposed to just represent the list of addons that come installed by `sb init`. If you want to use an addon or stories for an addon that is outside of that list (`links`, `essentials` and `interactions`), you need to pass `--addon a11y` to the sandbox script. We should probably do that in CI.

**Comment by tmeasday** [comment]
Addendum to the above, the `--addon` option doesn't actually work because `sb add` doesn't work, so for now you need to manually add the addon in `main.js` (and maybe `package.json`) to actually test this. 🙈

File: code/addons/a11y/template/stories/parameters.stories.ts
Comment (tmeasday) [inline code/addons/a11y/template/stories/parameters.stories.ts:42]: Is it obvious this is doing something? Would it be better to do these kind of things in the HTML-based story file?

**Review by tmeasday — APPROVED** [review]
This is pretty close, I just wonder if it might be better to move some of the parameter-based stories to the other file. @ndelangen I'll leave it to you to make the final call!

**Comment by ndelangen** [comment]
All right, i think I've improved things a bit, will merge when green!

**Pull request merged**
